src/App.tsx

```diff
@@ -1,14 +1,36 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { SelectedTodo } from './types/SelectedTodo';
+import { StatusTodo } from './types/StatusTodo';
+import { getFilteredTodos } from './helpers/TodoHelper';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<SelectedTodo>(null);
+  const [query, setQuery] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState<StatusTodo>(
+    StatusTodo.All,
+  );
+
+  useEffect(() => {
+    getTodos().then(setTodos);
+  }, []);
+
+  useEffect(() => {
+    const newFilteredTodos = getFilteredTodos(todos, selectedStatus, query);
+
+    setFilteredTodos(newFilteredTodos);
+  }, [todos, selectedStatus, query]);
```

Comment:
```suggestion
  const filteredTodos = getFilteredTodos(todos, selectedStatus, query);
```
Just store in a variable, if you want you can use `useMemo` but it's optional